RFC-011-SWAP-Omni-Layer-basic.md

```diff
@@ -0,0 +1,320 @@
+  "blocktime": 1556091362,
+  "positioninblock": 1,
+  "block": 441,
+  "confirmations": 6
+}
+```
+
+### Redeem transaction
+
+To redeem the HTLC, the redeemer MUST submit a valid Omni Layer transaction transferring all the omni tokens from the HTLC address to their desired address using simple send.
+The transaction should be constructed according to the [Constructing an Omni Layer Transaction](#constructing-an-omni-layer-transaction) section.
+To construct the witness data for the P2WSH output, the redeemer MUST use the procedure described in the [Redeem section of RFC005](./RFC-005-SWAP-Bitcoin-basic.md#redeem).
+
+Both parties SHOULD watch the blockchain for the a bitcoin transaction, that spends the HTLC output and is a valid Omni Layer transaction.
+Note that the redeemer MAY (but should not) use another type of Omni Layer transaction than simple send, hence monitoring MUST NOT be exclusive to simple send.
+
+For example, the following RPC call to omnicore can be used to verify the validity of a redeem transaction:
+- `txid`: the transaction id of the transaction (e.g. `ca30ac83bee95ea6191e74b96a8eae11d88eca8abbb55d688e0617a0b84f18d2`)
+- `referenceaddress`: the redeemer address (e.g. `mh9g3jCJxkc4tzV88THmQHGNGiCzUZ1zg6`)
+- `sendingaddress`: the HTLC address (e.g. `2N4CHfBpitHNXj9nBcJCFs7BXUaoNVTegBZ`)
+- `valid`: MUST be `true`
+- `amount`: the quantity of asset (e.g. `5300`)
+
+```
+$ omnicore-cli "omni_gettransaction" "ca30ac83bee95ea6191e74b96a8eae11d88eca8abbb55d688e0617a0b84f18d2"  
+{
+  "txid": "ca30ac83bee95ea6191e74b96a8eae11d88eca8abbb55d688e0617a0b84f18d2",
+  "fee": "0.00002500",
+  "sendingaddress": "2N4CHfBpitHNXj9nBcJCFs7BXUaoNVTegBZ",
+  "referenceaddress": "mh9g3jCJxkc4tzV88THmQHGNGiCzUZ1zg6",
+  "ismine": false,
+  "version": 0,
+  "type_int": 0,
+  "type": "Simple Send",
+  "propertyid": 2147483652,
+  "divisible": false,
+  "amount": "5300",
+  "valid": true,
+  "blockhash": "7e94a2a8eb188f2e08020ecea4aa30959525de2040fb75bc7911b3988f2ad7df",
+  "blocktime": 1556091006,
+  "positioninblock": 1,
+  "block": 442,
+  "confirmations": 5
+}
+```
+
+### Refund transaction
+
+To refund1 the HTLC, the funder MUST submit a valid Omni Layer transaction transferring all the omni tokens from the HTLC address to their desired address using simple send.
+The transaction should be constructed according to the [Constructing an Omni Layer Transaction](#constructing-an-omni-layer-transaction) section.
+To construct the witness data for the P2WSH output, the funder MUST use the procedure described in the [Redeem section of RFC005](./RFC-005-SWAP-Bitcoin-basic.md#redeem).
+
+Both parties SHOULD watch the blockchain for the a bitcoin transaction, that spends the HTLC output and is a valid Omni Layer transaction.
+Note that the funder MAY (but should not) use another type of Omni Layer transaction than simple send, hence monitoring MUST NOT be exclusive to simple send.
```

Comment:
Same as above, I would prefer removing the `(but should not)` part.

Reply:
same, I changed it all.